wallet/src/wallet.rs

```diff
@@ -540,13 +540,31 @@ fn process_show_vote_account(
         }
     );
     if !vote_state.votes.is_empty() {
-        println!("votes:");
-        for vote in vote_state.votes {
+        println!("recent votes:");
+        for vote in &vote_state.votes {
             println!(
                 "- slot={}, confirmation count={}",
                 vote.slot, vote.confirmation_count
             );
         }
+
+        // TODO: Use the real GenesisBlock from the cluster.
```

Comment:
Little dirty but I didn't feel like exposing a new GenesisBlock JSON RPC method just yet, and in the worse case if this is out of sync then the output `slots_in_epoch` variable will be wrong.   Annoying but not a big deal, and when this happens we can consider adding that new GenesisBlock JSON RPC method at that time :)